microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/core/PropertiesManagerForJava.java

```diff
@@ -268,10 +138,10 @@ private static ITypeRoot resolveTypeRoot(String uri, IJDTUtils utils, IProgressM
 		return unit != null ? unit : classFile;
 	}
 
-	private JDTQuarkusProjectInfo getJDTQuarkusProjectInfo(IJavaProject project) throws JavaModelException {
-		JDTQuarkusProjectInfo info = infos.get(project);
+	private JDTMicroProfileProject getJDTQuarkusProjectInfo(IJavaProject project) throws JavaModelException {

```

Comment:
getJDTMicroProfileProject if you want to stay consistent

Reply:
Fixed and move this method in a new class JDTMicroProfileProjectManager (more clean I find)